e2e/process-services-cloud/process-task-attach-content-file-cloud.e2e.ts

```diff
@@ -0,0 +1,160 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AlfrescoApiCompatibility as AlfrescoApi } from '@alfresco/js-api';
+import { browser } from 'protractor';
+import {
+    AppListCloudPage,
+    LoginSSOPage,
+    StartProcessCloudPage,
+    StringUtil,
+    TaskFormCloudComponent,
+    ProcessCloudWidgetPage,
+    ViewerPage,
+    UploadActions,
+    ContentNodeSelectorDialogPage
+} from '@alfresco/adf-testing';
+import { ProcessCloudDemoPage } from '../pages/adf/demo-shell/process-services/process-cloud-demo.page';
+import { NavigationBarPage } from '../pages/adf/navigation-bar.page';
+import { TasksCloudDemoPage } from '../pages/adf/demo-shell/process-services/tasks-cloud-demo.page';
+
+describe('Process Task - Attache content file', () => {
+
+    const loginSSOPage = new LoginSSOPage();
+    const navigationBarPage = new NavigationBarPage();
+    const appListCloudComponent = new AppListCloudPage();
+    const processCloudDemoPage = new ProcessCloudDemoPage();
+    const tasksCloudDemoPage = new TasksCloudDemoPage();
+    const taskFormCloudComponent = new TaskFormCloudComponent();
+    const processCloudWidget = new ProcessCloudWidgetPage();
+    const startProcessPage = new StartProcessCloudPage();
+    const contentNodeSelectorDialog = new ContentNodeSelectorDialogPage();
+    const viewerPage = new ViewerPage();
+
+    const processName = StringUtil.generateRandomString(10);
+    const simpleApp = browser.params.resources.ACTIVITI_CLOUD_APPS.SIMPLE_APP.name;
+    const folderName = StringUtil.generateRandomString(5);
+    let uploadedFolder: any;
+
+    const widgets = {
+        contentMultipleAttachFileId: 'UploadMultipleFileFromContentId'
```

Comment:
Can this value be saved into resources file?

Reply:
Updated